bfvmm/include/intrinsics/x86/intel/apic/x2apic.h

```diff
@@ -2231,6 +2409,12 @@ struct EXPORT_X2APIC x2apic_control final : public lapic_control {
     void write_icr(value_type icr) noexcept override
     { msrs::ia32_x2apic_icr::set(icr); }
 
+    void write_icr_low(value_type icr_low) noexcept override
```

Comment:
Lets go ahead and just remove the icr_low, high functions in all the classes. I think I'm going to go about this another way but we won't really need this for awhile